samples/tab-graph-toolkit/nodejs/src/index.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './components/App';
-import { Providers } from '@microsoft/mgt-element';
-import { Msal2Provider } from '@microsoft/mgt-msal2-provider';
-Providers.globalProvider = new Msal2Provider({
-  clientId: 'Your_Client_ID',
-  scopes: ['calendars.read', 'user.read', 'openid', 'profile', 'people.read', 'user.readbasic.all']
+import * as MicrosoftTeams from "@microsoft/teams-js";
+import { Providers } from '@microsoft/mgt';
+import { TeamsMsal2Provider } from '@microsoft/mgt-teams-msal2-provider';
+import './index.css';
+
+TeamsMsal2Provider.microsoftTeamsLib = MicrosoftTeams;
+
+Providers.globalProvider = new TeamsMsal2Provider({
+  clientId: '2c865108-9c5e-456c-94c3-bb792a77576b',
```

Comment:
Why you have hardcoded client id??
How are we going to get and update it, please check and update readme if needed.

Reply:
Actually I faced error on this line, while testing sample I was running app without updating client id here... because in readme it was not mentioned, by this error I came to know we need to pass client id.

I have updated In Readme as well.

Thanks for this comment.